src/lib/atoms/KeyValue/KeyValue.tsx

```diff
@@ -35,7 +64,7 @@ function KeyValue({ label, value, className, icon, appearance, ...restProps }) {
 
             <p
```

Comment:
I think the value and label better to render with check like .
Because now if there is no label passed the empty `<p>` renders.
```
{value && <p
                className={classnames(
                    'geneKeyValue__value ellipsis-text',
                    keyValueConfig.appearance[appearance]?.valueClassName
                )}
            >
                {value}
            </p>}
```

Reply:
after clearing partial this prop is made mandatory